src/pages/AnimeKO/main.ts

```diff
@@ -0,0 +1,82 @@
+import { pageInterface } from '../pageInterface';
+
+let jsonData;
+
+export const AnimeKO: pageInterface = {
+  name: 'AnimeKO',
+  domain: 'https://animeko.co',
+  languages: ['French'],
+  type: 'anime',
+  isOverviewPage() {
+    return !!j.$('.showcase-shadow').length;
+  },
+  isSyncPage() {
+    return !!j.$('.player-wrap').length;
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('.small-card h2').text().trim();
+    },
+    getIdentifier(url) {
+      return AnimeKO.overview!.getIdentifier(AnimeKO.sync.getOverviewUrl(url));
+    },
+    getOverviewUrl(url) {
+      return utils.absoluteLink(j.$('.small-card h2 a').attr('href'), AnimeKO.domain);
+    },
+    getEpisode(url) {
+      return Number(j.$('.player-releases :selected').text().split(' ')[1]);
+    },
+    nextEpUrl(url) {
+      if (j.$('.btn-next').length) {
+        return utils.absoluteLink(j.$('.btn-next').attr('href'), AnimeKO.domain);
+      }
+      return '';
+    },
+    getMalUrl(provider) {
+      if (jsonData.mal_id) {
+        return 'https://myanimelist.net/anime/' + jsonData.mal_id;
+      }
+      return false;
+    },
+  },
+  overview: {
+    getTitle(url) {
+      return j.$('.showcase h1').text().trim();
+    },
+    getIdentifier(url) {
+      return utils.urlPart(url, 4);
+    },
+    getMalUrl() {
+      if (jsonData.mal_id) {
+        return 'https://myanimelist.net/anime/' + jsonData.mal_id;
+      }
+      return false;
+    },
+    uiSelector(selector) {
+      j.$('.showcase');
```

Comment:
This does not inject the UI. It is a labeled wrong. Selector is the html that needs to be injected
EX.: https://github.com/MALSync/MALSync/blob/master/src/pages/Animeflv/main.ts#L38